src/plugins/intel_cpu/src/ngraph_transformations/mha_fusion.hpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ov {
+namespace intel_cpu {
+
+class MHAFusion: public ngraph::pass::MatcherPass {
```

Comment:
Minor:
There are four almost the same matcher passes here. Could we use here `GraphRewrite` to unite them as for [HSwish](https://github.com/openvinotoolkit/openvino/blob/186efc5125e20b8cd0676d15b4cac332eea7c8d3/src/common/transformations/include/transformations/common_optimizations/hswish_fusion.hpp#L68)?